projects/js-packages/components/components/loading-placeholder/style.module.scss

```diff
@@ -0,0 +1,18 @@
+@keyframes pulse {
+  0% {
+    opacity: .6;
+  }
+  50% {
+    opacity: 1;
+  }
+  100% {
+    opacity: .6;
+  }
+}
+
+.loadingPlaceholder {
```

Comment:
```suggestion
.placeholder {
```

There is no need to avoid naming collisions because this a css module file. The class names are hashed:

![2023-03-03_10-44-07](https://user-images.githubusercontent.com/8486249/222735462-c243543c-3c23-49b8-a1f2-74864205b6c6.png)